### PR TITLE
Add post on OMS rollback: INFODEV-2020

### PR DESCRIPTION
**Pull request by RackerKate**
In a comment on the pull request, include the following information and check off the list.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-184bb90262/)

**Comment by RackerKate** [comment]
@stephamon Please review.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-d943377ed5/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-e9da8caa51/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-e9da8caa51/)

File: _posts/2018-08-20-oms-rollback.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+You can use any of the following methods to restore the repository database
+from the backup:
+
+- RMAN backup restoration and recovery
+- Export and import
+- Oracle Data Guard with broker
+- Cold backup
+
+#### Phase 2: Move OMS and agents to a new host
```
Comment (stephamon) [inline _posts/2018-08-20-oms-rollback.md]: This should be the same header level as phase 1

File: _posts/2018-08-20-oms-rollback.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+This section describes how to complete phase two.
```
Comment (stephamon) [inline _posts/2018-08-20-oms-rollback.md]: Given the section title, I'd phrase this as something like "This section describes how to move OMS and agents to a new host"

File: _posts/2018-08-20-oms-rollback.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+2. Use the backup configuration and the following OMSCA command to
+   recreate the OMS:  
+
+       oms:/u02/app/oracle/Middleware/oms:N
+       REPDB:/u01/app/oracle/product/12.1.0.2/DB_1:N
+
+       [oracle@oem251 ~]$ omsca recover -as -ms -nostart -backup_file /ora_global_nfs/BACKUP/REPDB_BACKUP/opf_ADMIN_20160303_105032.bka
+
+### Phase 3
```
Comment (stephamon) [inline _posts/2018-08-20-oms-rollback.md]: Is there a subtitle for this one, like the previous two phases? Maybe something like "Phase 3: Configure the new host"? You might want to ask when you send this back to Smriti and the author for their final review

**Comment by stephamon** [comment]
Just a couple things @RackerKate

File: _posts/2018-08-20-oms-rollback.md (hunk 1 above)
Comment (RackerKate) [reply _posts/2018-08-20-oms-rollback.md]: Got it, thanks.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-c1c37c9c11/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-6e67bbbb08/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-6e67bbbb08/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-43479a8c15/). :bow:

**Comment by RackerKate** [comment]
@catlook For your review please.

**Comment by catlook** [comment]
on it

**Pull request merged**